### ISSUE TITLE
Custom metrics for Beit3

### ISSUE THREAD

**Issue by eliorkfacetrom** [microsoft/unilm#1090]
I fine-tuned Beit3 for my image classification task. The most interesting parts for my research are the evaluation metrics (AUC, Precision, Recall, etc).

I find it kind of hard to save the logits of the val data for each step (maybe because of my lack of knowledge with metric_logger) and aggregate the results in the after_eval step.

Any help with that would be appreciated.

**Comment by eliorkfacetrom** [comment]
Solved manually.
